MobileSDK/sdk/build.gradle

```diff
@@ -0,0 +1,27 @@
+apply plugin: 'android-library'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion "23.0.2"
+
+    defaultConfig {
+        minSdkVersion 15
+        targetSdkVersion 23
+        versionCode 1
+        versionName "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
```

Comment:
Are we going to add back all the publishing options/commands for uploading artifacts to Maven Central in a separate PR, or are we getting rid of all that for some reason? The whole point of a separate SDK project was to turn Lantern into an Android library our own apps ("Lantern Android", FireTweet) and third party apps could integrate easily http://lantern.readthedocs.org/en/latest/ Just clarifying.